### PR TITLE
fix: Ensure escoria#apply_settings only called in game not editor

### PR DESCRIPTION
**Pull request by dploeger**
Found this during the stream and was annoyed. 😉

### CODE REVIEW

**Review by BHSDuncan — APPROVED** [review]

File: addons/escoria-core/game/escoria.gd [hunk 1]
```diff
@@ -179,31 +179,32 @@ func new_game():
...
-	AudioServer.set_bus_volume_db(
-		AudioServer.get_bus_index(BUS_MUSIC),
-		linear2db(settings.music_volume)
-	)
-	AudioServer.set_bus_volume_db(
-		AudioServer.get_bus_index(BUS_SPEECH),
-		linear2db(settings.speech_volume)
-	)
-	TranslationServer.set_locale(settings.text_lang)
+	if !Engine.is_editor_hint():
```
Comment (balloonpopper) [inline addons/escoria-core/game/escoria.gd:182]:
```suggestion
	if not Engine.is_editor_hint():
```

**Review by balloonpopper — CHANGES_REQUESTED** [review]
Added comment to change ! to "not" as per today's stream

**Review by StraToN — APPROVED** [review]

**Comment by StraToN** [comment]
Thanks!

**Comment by dploeger** [comment]
Oh, wonderful. You already fixed it! Thanks a lot. And I just wondered what I forgot to do after the stream! 😆

**Pull request merged**